src/website/entrypoint.sh

```
@@ -3,15 +3,33 @@
 # Exit immediately if a command exits with a non-zero status
 set -e
 
+# Function to handle signals and gracefully shut down Gunicorn
+_term() {
+  echo "Caught SIGTERM signal! Shutting down Gunicorn..."
+  kill -TERM "$child" 2>/dev/null
```

Shellcheck (0.10.0)

[warning] child is referenced but not assigned.

(SC2154)